templates/faq.html

```diff
@@ -239,7 +239,7 @@ <h1 class="white-text">Hello, how can we help you?</h1>
                                 <!-- faq item -->
                                 <div class="panel panel-default">
                                     <div role="tablist" id="type1-headingFour" class="panel-heading">
-                                        <a data-toggle="collapse" data-parent="#collapse-three" href="#collapse-three-link9"><h4 class="panel-title">Will I receive a certificate at the end of the KWoC 2016?<span class="pull-right"><i class="fa fa-plus"></i></span></h4></a>
+                                        <a data-toggle="collapse" data-parent="#collapse-three" href="#collapse-three-link9"><h4 class="panel-title">Will I receive a certificate at the end of the KWoC 2017?<span class="pull-right"><i class="fa fa-plus"></i></span></h4></a>
```

Comment:
As @harishnandan points out, even I think it would be good if we just keep it at the end of `kwoc program`.